code/game/objects/structures/dropship_equipment.dm

```diff
@@ -295,64 +295,180 @@
 	else
-		icon_state = "flare_system"
+		icon_state = "nade_system"
+
 
-///gets target and deploy the flare launcher
-/obj/structure/dropship_equipment/shuttle/flare_launcher/proc/deploy_flare()
+
+//////////////////////////////////// tanglefoot emitter //////////////////////////////////////
+
+/obj/structure/dropship_equipment/shuttle/tangle_emitter
+	equip_category = DROPSHIP_WEAPON
+	name = "disposable tanglefoot emitter"
+	desc = "A system that can emit tanglefoot while landing the aircraft to support aggressive landing positions. Can only be used once every ten minutes. Fits on the weapon attach points of dropships. You need a powerloader to lift it."
+	icon_state = "tfoot_system"
+	point_cost = 150
+	dropship_equipment_flags = IS_INTERACTABLE
+	/// Whether the system is currently enabled to activate on landing or not
+	var/enabled = TRUE
+	/// What type of smoke to use
+	var/obj/item/explosive/grenade/smokebomb/drain/pellet/pellet_type
+	/// Cooldown for emitting smoke
+	var/cooldown_length = 10 MINUTES
+	/// Identifier for system feedback
+	var/id = 0
+	COOLDOWN_DECLARE(use_cooldown)
+
+
+/obj/structure/dropship_equipment/shuttle/tangle_emitter/Initialize(mapload)
+	. = ..()
+	id = UNIQUEID
+
+/obj/structure/dropship_equipment/shuttle/tangle_emitter/equipment_interact(mob/user)
+	if(!enabled)
+		enabled = TRUE
+		update_icon()
+		user.balloon_alert(user, "You activate [src] #[id]. It will now deploy smoke when landing.")
+		RegisterSignal(linked_shuttle, COMSIG_MINIDROPSHIP_BEGIN_LANDING, PROC_REF(drop_pellet_to_location))
+		return
+	enabled = FALSE
+	update_icon()
+	user.balloon_alert(user, "You disable [src] #[id]. It will no longer deploy smoke when landing.")
+	UnregisterSignal(linked_shuttle, COMSIG_MINIDROPSHIP_BEGIN_LANDING)
+
+/obj/structure/dropship_equipment/shuttle/tangle_emitter/update_equipment()
+	. = ..()
+	if(ship_base)
+		setDir(ship_base.dir)
+		if(enabled)
+			update_icon()
+			RegisterSignal(linked_shuttle, COMSIG_MINIDROPSHIP_BEGIN_LANDING, PROC_REF(drop_pellet_to_location))
+	else
+		setDir(initial(dir))
+	update_icon()
+
+
+
+/obj/structure/dropship_equipment/shuttle/tangle_emitter/update_icon_state()
+	. = ..()
+	if(ship_base)
+		if(COOLDOWN_CHECK(src, use_cooldown))
+			icon_state = "tfoot_system_installed"
+			if(enabled)
+				icon_state = "tfoot_system_enabled"
+		else
+			icon_state = "tfoot_system_empty"
+	else
+		icon_state = "tfoot_system"
+
+/// Sets up and activates smoke effect
+/obj/structure/dropship_equipment/shuttle/tangle_emitter/proc/drop_pellet_to_location(datum/source, turf/landing_spot)
+	if(!istype(linked_console, /obj/machinery/computer/camera_advanced/shuttle_docker/minidropship))
+		return
+	var/obj/machinery/computer/camera_advanced/shuttle_docker/minidropship/console = linked_console
+	if(console.next_fly_state != SHUTTLE_ON_GROUND || !enabled || !landing_spot)
+		return
+	if(!COOLDOWN_CHECK(src, use_cooldown))
+		console.say("Emitter system #[id] recharging. Unable to deploy smoke.")
```

Comment:
balloon alert?

Reply:
No, system would alert all crew, not just the pilot